liubai-backends/liubai-laf/cloud-functions/open-connect.ts

```
@@ -66,10 +77,461 @@ export async function main(ctx: FunctionContext) {
+      })
+      return Boolean(domain)
+    }
+    catch(err) {
+      console.warn("_checkWebhookUrl error: ", err)
+    }
+    return false
+
+  }
+
+  static isWpsWebhookUrl(link: string) {
+    const list = this.wpsDomains
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)